packages/functions/src/sendEmail/https.ts

```diff
@@ -90,7 +91,7 @@ export const sendEmail = functions
 
       // Construct an email for process delivery
       const email = {
-        from: emailFrom,
+        from: emailNameFrom || emailFrom,
```

Comment:
I think we should combine name and email here in the form of a "mailbox": https://superuser.com/questions/1625686/what-is-the-email-address-format-name-email-called

Reply:
Oook...had the same idea, but decided to cut it as I wasn't sure if it was my overcomplicating things (plus I didn't know the standard format, so thanks for this...)